nrfx/hal/nrf_auxpll.h

```diff
@@ -123,6 +123,27 @@ typedef struct
     nrf_auxpll_divider_range_t range; /**< Loop divider base settings */
 } nrf_auxpll_config_t;
 
+/**
+ * @brief AUXPLL static configuration initializer.
+ * 
+ * @param out_drive Output buffer drive strength selection. Range 0..3.
+ * @param constant_i Constant current tune for ring oscillator. Range 0..15.
+ * @param sdm_off Turn off sigma delta modulation.
+ * @param dither_off Turn off dither in sigma delta modulator.
+ * @param range Loop divider base settings.
+ */
+#define AUXPLL_CFGSTATIC_INIT(out_drive, constant_i, sdm_off, dither_off, range) \
```

Comment:
nit: perhaps it would be good to use here parameter names matching the corresponding fields in `nrf_auxpll_config_t`